tools/godotcpp.py

```diff
@@ -337,6 +338,10 @@ def options(opts, env):
     opts.Add(BoolVariable("debug_symbols", "Build with debugging symbols", True))
     opts.Add(BoolVariable("dev_build", "Developer build with dev-only debugging code (DEV_ENABLED)", False))
     opts.Add(BoolVariable("verbose", "Enable verbose output for the compilation", False))
+    opts.Add(
+        "cache_path", "Path to a directory where SCons cache files will be stored. No value disables the cache.", ""
+    )
+    opts.Add("cache_limit", "Max size (in GiB) for the SCons cache. 0 means no limit.", "0")
```

Comment:
If a variable has a unit, I like to add it to the end of the variable name (`cache_limit_gb` / `cache_limit_gib`) to ensure its unit will [never be misinterpreted](https://en.wikipedia.org/wiki/Mars_Climate_Orbiter). I don't think it's common throughout anything Godot though, but I'd still advise to do it.

Reply:
I'll add those as aliases; that's a change many options could benefit from